src/components/Calculator.js

```diff
@@ -1,33 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
 
-const Calculator = () => (
-  <section className="calculator-section">
-    <div className="answer-section">
-      <span>0</span>
-    </div>
+const Calculator = () => {
+  const [details, setDetails] = useState({ total: '0', next: null, operation: null });
 
-    <div className="buttons-section">
-      <span className="button">Ac</span>
-      <span className="button">+/=</span>
-      <span className="button">%</span>
-      <span className="button operator">/</span>
-      <span className="button">7</span>
-      <span className="button">8</span>
-      <span className="button">9</span>
-      <span className="button operator">x</span>
-      <span className="button">4</span>
-      <span className="button">5</span>
-      <span className="button">6</span>
-      <span className="button operator">-</span>
-      <span className="button">1</span>
-      <span className="button">2</span>
-      <span className="button">3</span>
-      <span className="button operator">+</span>
-      <span className="button item17">0</span>
-      <span className="button">.</span>
-      <span className="button operator">=</span>
-    </div>
-  </section>
-);
+  const { total, next, operation } = details;
+
+  const makeCalculation = (btn) => {
+    setDetails((details) => calculate(details, btn.target.textContent));
+  };
+
+  return (
+    <section className="calculator-section">
+      <div className="answer-section">
+        <span>
+          {total}
+          {next}
+          {operation}
+        </span>
+      </div>
+
+      <div className="buttons-section">
+        <span className="button" onClick={makeCalculation}>Ac</span>
```

Comment:
- [ ] This button is expected to clear current figures and reset the calculator but it's not working as it's supposed to. Kindly watch [this video](https://www.loom.com/share/ccb913207f044b7c8a22edc7784655ca) to see a demonstration of areas that need fixing 👍🏽

Reply:
Hey @AmaduKamara  Thank you for your feedback. Let me work on the changes

Reply:
I have now implemented the changes. Thank you once again.

Reply:
Good job @Fene-87 👍🏽